formatting results for meta data service

Aishwarya020 opened the pull request.

jgoldverg reviewed (changes requested):
1. Lets change the read & write count back to Integer. Use the class Integer and not the primitive type maybe? I would vote primitve but do what u can.
2. In JobController.java you are importing all of the collections library. Can u add the separate imports manually
3. So I see you have converted to using the Set class. Would this break if there are duplicate requests? 
4. Instead of writing the empty constructors you can use the Lombok @NoArgsConstructs in the class to generate that constructor. It looks much cleaner :)

Aishwarya020 commented:
1. Changed Read Count and Write Count to Set<Integer>
2. JobController refactored with specific imports
3. Set class is being used to avoid repetition of details pertaining to the same job request.
4. Added Lombok @NoArgsConstructs

Aishwarya020 commented:
=-0821-0987654321q`
